jupyterhub_moss/form/option_form.js

```diff
@@ -314,19 +314,42 @@ function setSimplePartition(name) {
   }
 }
 
+function updateMemValue() {
+  const memHiddenElem = document.getElementById('mem');
+  const memInputElem = document.getElementById('mem_input');
+  const value = memInputElem.value;
+
+  // Pass empty field and 0 as is, append G for others
+  memHiddenElem.value = value && value !== '0' ? `${value}G` : value;
```

Comment:
Why a specific case for `0` ?

Reply:
It's a special case meaning all the memory.

Reply:
I see. Note that `value && value !== 0` is the same as `value` as `0` is falsy

Reply:
Yes, this what I wrote first, but `'0' !== 0` is `false